examples/BoxPlot.elm

```diff
@@ -0,0 +1,312 @@
+            ]
+            []
+         , rect
+            [ x indxXLeft
+            , y thirdQuantileY
+            , width boxWidth
+            , height (firstQuantileY - thirdQuantileY)
+            , fill <| Reference "linGradientDuo"
+            , opacity <| Opacity 0.8
+            ]
+            []
+         , line
+            -- median line in middle of rectangle
+            [ x1 indxXLeft
+            , y1 serieMedianY
+            , x2 <| indxXLeft + boxWidth
+            , y2 serieMedianY
+            , strokeWidth 1
+            , stroke <| Paint <| Color.black
+            , opacity <| Opacity 0.6
+            ]
+            []
+         ]
+            ++ List.map outlierCircle outliersXY
+        )
+
+
+outlierCircle : ( Float, Float ) -> Svg msg
+outlierCircle ( x, y ) =
+    circle
+        [ cx x
+        , cy y
+        , r 3
+        , fill <| Paint <| Color.rgb255 180 20 20
+        , strokeWidth 0
+        , stroke <| PaintNone
+        , opacity <| Opacity 0.7
+        ]
+        []
+
+
+yGridLine : Int -> Float -> Svg msg
+yGridLine index tick =
+    line
+        [ x1 0
+        , x2 xAxisWidth
+        , y1 (Scale.convert yScale tick)
+        , y2 (Scale.convert yScale tick)
+        , stroke <| Paint Color.black
+        , strokeWidth (Basics.max (toFloat (modBy 2 index)) 0.5)
+        , opacity <| Opacity 0.4
+        ]
+        []
+
+
+
+-- rectangle for colored background
+-- bgRect : Svg msg
+-- bgRect =
+--     rect [ x 0
+--             , y 0
+--             , width <| xAxisWidth
+--             , height <| yAxisHeight
+--             , fill <| Paint <| Color.red
+--             , opacity <| Opacity 0.2
+--             ]
+--             []
+
+
+myDefs : List (Svg msg)
+myDefs =
+    [ linearGradient
+        [ id "linGradientDuo"
+        , TypedSvg.Attributes.x1 <| Percent 0.0
+        , TypedSvg.Attributes.y1 <| Percent 0.0
+        , TypedSvg.Attributes.x2 <| Percent 0.0
+        , TypedSvg.Attributes.y2 <| Percent 100.0
+        ]
+        [ stop [ offset "0%", stopColor "#e52d27" ] []
+        , stop [ offset "100%", stopColor "#b31217" ] []
+        ]
+    ]
+
+
+view : List (List Float) -> Svg msg
+view model =
+    svg [ viewBox 0 0 w h ]
+        [ defs [] myDefs
+
+        -- enable for colored plot background
+        -- , g [ transform [ Translate padding padding ], class [ "bg-rect" ] ] <|
+        --     [ bgRect ]
+        , g [ transform [ Translate padding (padding + 0.5) ] ] <| List.indexedMap yGridLine <| Scale.ticks yScale 9
+        , g [ transform [ Translate (padding - 1) (h - padding) ] ]
+            [ xAxis model ]
+        , g [ transform [ Translate (padding - 1) padding ] ]
+            [ yAxis ]
+        , g [ transform [ Translate padding padding ], class [ "series" ] ] <|
+            List.indexedMap (\indx lst -> column (xScale model) lst indx) model
+        ]
+
+
+main : Svg msg
+main =
+    view dataSeries
+
+
+dataSeries : List (List Float)
+dataSeries =
```

Comment:
I think this example would be a bit more interesting if the data was a bit less arbitrary looking. As in if we had some nice labels for the x axis rather than just 1,2,3...